app/translations/en.json

```
@@ -993,6 +995,18 @@
                     "label": "Sort by name",
                     "description": "If set to true, then blueprints will be sorted by name."
                 }
+            },
+            "authenticationWarning": "No GitHub credentials provided! Widget may stop working after reaching unrestricted query limit (~50). Fix this by adding &github-username& and &github-password& entries to your secrets store (Secrets widget).",
+            "uploadingMessage": {
+                "uploading": "Uploading {{blueprintName}} blueprint",
+                "redirection": "After completing the upload, you'll be redirected to the dedicated blueprint page"
```

I'm not sure about the message, maybe something like: 
```suggestion
                "redirection": "Once the blueprint is uploaded, you'll be redirected to the blueprint deployment form."
```
?

I'm also not sure about my proposal. Probably somebody from the product team should validate it.

---

This approach seems to be more precise 💪 
I've just got one concern about the `you'll be redirected to the blueprint deployment form`, as the user will be redirected to a different page upon opening which he'll be able to see the mentioned modal
Once closing the modal, he'll be on a different page than he was while submitting the form

Because of that I think that there may be a room for improvement for this part of the message

But as you've suggested, I'll just contact product team to not wonder around, but just to get straight answers 🚀 

Thanks for the suggestion! 🎉 
(first part of it is really neat! 😎)

---

The designer is still not responding and because of that Ilan decided, that for now we can put the following message to the code:

```typescript
Uploading '<blueprint name>' blueprint...
Once the blueprint is uploaded, you'll be redirected to the blueprint deployment page.
```

So I'll do just that 😉